### PR TITLE
Reduce scope to annual CO2 update and PlanetIQ assimilation

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

The `obsproc/v1.3` upgrade is delayed, therefore, the `GFSv16.3.20` will now update the following:

- install 2025 carbon dioxide (CO2) fix file updates before the start of the new year
- updated GSI global_convinfo.txt to turn on assimilation of PlanetiQ GPS Radio Occultation (GPS-RO) data
- memory increases for a number of jobs (provided by NCO)
- load the cmdaccel module at the start of wave point postprocessor jobs to stabilize runtime (provided by NCO)

FYI @DianeStokes-NCO 

Refs #2558

# Type of change

Production update

# How has this been tested?

NCO para testing

### CODE REVIEW

**Comment by KateFriedman-NOAA** [comment]
Awaiting final confirmation on whether an additional GSI code change will be included.

**Review by aerorahul — APPROVED** [review]
looks ok to me.
quite an update to the memory requests.  Can they tell us what changed to get the new estimates? Should note it in the release notes for documentation that they came from testing by the SPA team.

**Comment by KateFriedman-NOAA** [comment]
Folded in a new GSI tag and additional memory increase, with mention in release notes. Awaiting potential final resource change to a walltime from NCO but otherwise this PR should be reviewed and approved. Thanks!

**Comment by KateFriedman-NOAA** [comment]
Added increase to enkfgdas_update job walltime (35mins -> 40mins) @ [7552cf4](https://github.com/NOAA-EMC/global-workflow/pull/3144/commits/7552cf44da6893b1b8951beaf2603e0edb256547). Update was merged into ops this morning, so this is a complete changeset now. Please review and approve, thanks!

File: versions/run.ver [hunk 1]
```diff
@@ -48,4 +48,6 @@ export zlib_ver=1.2.11
 export sp_ver=2.3.3
 export ip_ver=3.3.3
 
+export cmdaccel_ver=1.1
```
Comment (WalterKolczynski-NOAA) [inline versions/run.ver:51]: What is this new module?

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by DianeStokes-NCO** [comment]
Here is the description of the cmdaccel module:

------------------------------------------------------------
--------------------------------------------
   /apps/prod/lmodules/sys_core/cmdaccel/1.1.lua:
------------------------------------------------------------
--------------------------------------------
Description:
Sets up environment-modifying functions for the most frequently used
scripting commands from /usr/bin.
These functions adjust the LD_LIBRARY_PATH environment variable temporarily
while running the command to
avoid unnecessary searches through application paths in /apps, potentially
saving considerable execution
time.

In cmdaccel/1.1, functions are created for the following commands:
  awk  cat  chmod  chown  cp  cut  date  env  grep  ln  mkdir  rm  sed
tar  wc
The function for cut, eg, looks like:
function cut() { LD_PRELOAD= LD_LIBRARY_PATH="/lib64:/usr/lib64"
/usr/bin/cut "$@"; }

The gfs_wave_post*pnt* jobs run many small scripts in parallel and those
scripts run many basic commands.   We see wall clock failures for thes jobs
too often because, for some reason *(system load, slow node, etc),* they
run much more slowly than usual.  Additionally, runtime for these jobs have
increased since the SP4 upgrade.  GDIT has made several suggestions to
improve performance of these jobs.  Some require script or source code
changes.  One simple suggestion that provided obvious improvement was to
use the cmdaccel module.

Following are typical run times in minutes before and after the SP4 upgrade
and after SP4 when we load cmdaccel/1.1:
           *job*             *SP3*   *SP4*   *SP4 w/cmdaccel*
  jgfs_wave_post_bndpnt     39    45    30 minutes
  jgfs_wave_post_bndpntbll  34    36    22 minutes
  jgfs_wave_postpnt         48    57    40 minutes

I also added the load of cmdaccel to jgdas_wave_postpnt: 1) for consistency
with jgfs_wave_postpnt; and 2) because it helped rather than hurt.  Typical
runtimes for that job:
           *job*             *SP3*   *SP4*   *SP4 w/cmdaccel*
  jgdas_wave_postpnt       1.6   2.3   1.8 minutes


I compared output from runs with and without cmdaccel and content of the
output files matched.
Please let me know if you have further questions or concerns.
    Diane

File: versions/run.ver (hunk 1 above)
Comment (KateFriedman-NOAA) [reply versions/run.ver:51]: See Diane's comment https://github.com/NOAA-EMC/global-workflow/pull/3144#issuecomment-2533190930

**Comment by KateFriedman-NOAA** [comment]
Thanks @DianeStokes-NCO for the description on what the `cmdaccel` module is and how it helps.

**Comment by aerorahul** [comment]
Thanks @DianeStokes-NCO for the explaination.

**Pull request merged**